x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/search_indices.tsx

```diff
@@ -43,93 +55,99 @@ export const baseBreadcrumbs = [
 ];
 
 export const SearchIndices: React.FC = () => {
-  const { initPage, searchEnginesLoadSuccess, searchIndicesLoadSuccess } =
-    useActions(SearchIndicesLogic);
-  const { searchIndices, searchEngines } = useValues(SearchIndicesLogic);
+  const { makeRequest, onPaginate } = useActions(IndicesLogic);
+  const { meta, indices, isLoading } = useValues(IndicesLogic);
+
+  const [calloutDismissed, setCalloutDismissed] = useLocalStorage<boolean>(
+    'enterprise-search-indices-callout-dismissed',
+    false
+  );
 
   useEffect(() => {
-    initPage();
-  }, []);
-
-  // TODO This is for easy testing until we have the backend, please remove this before the release
-  // @ts-ignore
-  window.contentActions = {
-    initPage,
-    searchIndicesLoadSuccess,
-    searchEnginesLoadSuccess,
-  };
-
-  // TODO: Replace with a real list of indices
+    makeRequest({ meta });
+  }, [meta.page.current]);
+
   const columns = [
     {
       field: 'name',
       name: i18n.translate('xpack.enterpriseSearch.content.searchIndices.name.columnTitle', {
-        defaultMessage: 'Search index name',
+        defaultMessage: 'Index name',
       }),
-      sortable: true,
-      truncateText: true,
-      render: (name: string, { indexSlug }: SearchIndex) => (
+      render: (name: string) => (
         <EuiLinkTo
           data-test-subj="search-index-link"
-          to={generatePath(SEARCH_INDEX_PATH, { indexSlug })}
+          to={generatePath(SEARCH_INDEX_PATH, { indexName: encodeURIComponent(name) })}
```

Comment:
Ah, we dropped the encode/decodeURI everywhere because we found out that we don't need it. Index names are always URI-compatible.